docs/usage/configuration-options.md

```diff
@@ -2123,6 +2123,16 @@ Currently this applies to the `minimumReleaseAge` check only.
 The `flexible` mode can result in "flapping" of Pull Requests, for example: a pending PR with version `1.0.3` is first released but then downgraded to `1.0.2` once it passes `minimumReleaseAge`.
 We recommend that you use the `strict` mode, and enable the `dependencyDashboard` so that you can see suppressed PRs.
 
+## keepUpdatedLabel
+
+On supported platforms it is possible to add a label to a PR to recreate/rebase it when the branch falls
+1 or more commits behind its base branch. Adding the `keepUpdatedLabel` label to a PR makes Renovate behave
+as if `rebaseWhen` were set to `behind-base-branch`, but only for the given PR. The label is not removed from the PR after the rebase is complete, like what happens with `rebaseLabel`.
+
+This can be useful when you have approved certain PRs and want to keep them updated until they are ready
+to be merged. The setting `keepUpdatedLabel` is best used in conjunction with `rebaseWhen` set to the
+values of `never` or `conflicted` that limit rebasing.
```

Comment:
```suggestion
On supported platforms it is possible to add a label to a PR to recreate/rebase it when the branch falls 1 or more commits behind its base branch.
Adding the `keepUpdatedLabel` label to a PR makes Renovate behave as if `rebaseWhen` were set to `behind-base-branch`, but only for the given PR.
The label is not removed from the PR after the rebase is complete, unlike what happens with `rebaseLabel`.

This can be useful when you have approved certain PRs and want to keep them updated until they are ready to be merged.
The setting `keepUpdatedLabel` is best used in conjunction with `rebaseWhen` set to the values of `never` or `conflicted` that limit rebasing.
```